src/librustdoc/html/render/mod.rs

```diff
@@ -1952,11 +1943,14 @@ fn sidebar_deref_methods(cx: &Context<'_>, out: &mut Buffer, impl_: &Impl, v: &V
             .iter()
             .filter(|i| i.inner_impl().trait_.is_some())
             .any(|i| i.inner_impl().trait_.def_id_full(c) == c.deref_mut_trait_did);
-        let inner_impl = target
-            .def_id_full(c)
+        let inner_impl = target.def_id_full(c);
+        debug!("target.def_id_full: {:?}", inner_impl);
```

Comment:
(sigh, I wonder if this extra debugging lines were part of why this didn't backport cleanly...)